turbo.json

```diff
@@ -37,6 +37,7 @@
     "NEXT_PUBLIC_IPFS_GATEWAY",
     "NEXT_PUBLIC_SENTRY_DSN",
     "NEXT_PUBLIC_ZORA_API_KEY",
+    "NEXT_FARCASTER_HUB",
```

Comment:
do we need this exposed in the browser? if that's the case you need to prefix with `NEXT_PUBLIC` otherwise you don't need the NEXT prefix if it's just needed server side. 

https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser

Reply:
Yep. I can do that.